extensions/vscode/src/api/types/contentRecords.ts

```diff
@@ -45,7 +45,7 @@ type ContentRecordRecord = {
   saveName: string;
   createdAt: string;
   localId: string;
```

Comment:
The `localId` should also be removed. I'm not seeing it in the response any longer with the removal.

Reply:
Good point, thanks for finding these!